.pre-commit-config.yaml

```diff
@@ -8,8 +8,12 @@ repos:
       - id: end-of-file-fixer
       - id: mixed-line-ending
       - id: trailing-whitespace
-
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.0
+    rev: 3.9.1
     hooks:
       - id: flake8
+  - repo: https://github.com/awslabs/cfn-python-lint
+    rev: v0.48.3
+    hooks:
+      - id: cfn-python-lint
+        files: .*\.(json|yml|yaml)$
```

Comment:
Since we're using the `.cfnlintrc.yaml` file this is probably redundant here (maybe it will fix the circleci issue?). Other route is to move the ignore_templates to pre-commit's excludes instead. If I recall `pre-commit` passes one file at a time to the linter tool so filtering in the tool's config is redundant with what pre-commit is doing.